packages/redux/src/returns/reducer.ts

```diff
@@ -30,11 +34,17 @@ const error = (state = INITIAL_STATE.error, action: AnyAction) => {
     case actionTypes.FETCH_RETURN_PICKUP_CAPABILITIES_FAILURE:
     case actionTypes.FETCH_RETURN_FAILURE:
     case actionTypes.UPDATE_RETURN_FAILURE:
+    case actionTypes.FETCH_RETURN_PICKUP_RESCHEDULE_REQUEST_FAILURE:
+    case actionTypes.FETCH_RETURN_PICKUP_RESCHEDULE_REQUESTS_FAILURE:
+    case actionTypes.CREATE_RETURN_PICKUP_RESCHEDULE_REQUEST_FAILURE:
       return action.payload.error;
     case actionTypes.CREATE_RETURN_REQUEST:
     case actionTypes.FETCH_RETURN_PICKUP_CAPABILITIES_REQUEST:
     case actionTypes.FETCH_RETURN_REQUEST:
     case actionTypes.UPDATE_RETURN_REQUEST:
+    case actionTypes.FETCH_RETURN_PICKUP_RESCHEDULE_REQUEST_REQUEST:
+    case actionTypes.FETCH_RETURN_PICKUP_RESCHEDULE_REQUESTS_REQUEST:
+    case actionTypes.CREATE_RETURN_PICKUP_RESCHEDULE_REQUEST_REQUEST:
```

Comment:
There is a bug here that I fixed in another PR but you may include as well. This is missing a return statement to clear the error for these action types:
`return INITIAL_STATE.error`

Reply:
done!